Composer/packages/client/src/router.tsx

```diff
@@ -106,8 +120,24 @@ const projectStyle = css`
 const ProjectRouter: React.FC<RouteComponentProps<{ projectId: string; skillId: string }>> = (props) => {
   const { projectId = '' } = props;
   const schemas = useRecoilValue(schemasState(projectId));
-  const { fetchProjectById } = useRecoilValue(dispatcherState);
+  const { fetchProjectById, setSettingStateWithoutSync } = useRecoilValue(dispatcherState);
   const botProjects = useRecoilValue(botProjectIdsState);
+  const botProjectsMetaData = useRecoilValue(botProjectSpaceSelector);
```

Comment:
nit: Maybe name this botProjectSpaceData? since that selector gives way more than the metadata of the bots.